qiskit/qpy/formats.py

```diff
@@ -155,8 +154,49 @@
 CUSTOM_CIRCUIT_INST_DEF_PACK = "!H1cII?Q"
 CUSTOM_CIRCUIT_INST_DEF_SIZE = struct.calcsize(CUSTOM_CIRCUIT_INST_DEF_PACK)
 
+# CALIBRATION
+CALIBRATION = namedtuple("CALIBRATION", ["num_cals"])
+CALIBRATION_PACK = "!I"
```

Comment:
I'm not sure how many calibrations are typical on a circuit, this sets the upper bound at `2**32` which is probably sufficient but I'm wondering if we want to allow more or less.